package/network/services/hostapd/files/hostapd.sh

```diff
@@ -752,7 +752,8 @@ hostapd_set_bss_options() {
 		}
 	}
 
-	append bss_conf "ssid=$ssid" "$N"
+	[ "x${ssid}" = 'x' ] || append bss_conf "ssid=${ssid}" "$N"
+	[ "x${ssid2}" = 'x' ] || append bss_conf "ssid2=${ssid2}" "$N"	
```

Comment:
Having both values defined doesn't "hurt", `hostapd` just uses the latter. LuCI however can only show the former. I'm wondering if there is a clean implementation for that.

Reply:
Yes, handling that in LuCI at present is a weak point and is basically extra work. For a cheap alternative, can just probably mark the `ssid2` parameter as "console / `uci` only" ?

Reply:
I'll come up with something since you plan to drop author ship anyway :)